### DISCUSSION TITLE
Question about the folder structure in the contribution.md file

### DISCUSSION THREAD

**Discussion by ChinmayMhatre** [Ashutosh00710/github-readme-activity-graph#63] [Q&A] [answered]
Could you tell me how did you make the folder structure in the contribution.md file . I am unable to find the symbols used in the section . Would appreciate it.

**Comment by Ashutosh00710** [comment, marked as answer]
I got these symbols to build the folder structure from [here](https://github.com/kriasoft/Folder-Structure-Conventions/blob/master/README.md/) (I used the one mentioned in the **`Documentation Files`** Section)

You can copy from there and paste it inside this

\```md 
.
├── ...
├── docs                    # Documentation files (alternatively `doc`)
│   ├── TOC.md              # Table of contents
│   ├── faq.md              # Frequently asked questions
│   ├── misc.md             # Miscellaneous information
│   ├── usage.md            # Getting started guide
│   └── ...                 # etc.
└── ...
\```

You will see the result like this

```md
.
├── ...
├── docs                    # Documentation files (alternatively `doc`)
│   ├── TOC.md              # Table of contents
│   ├── faq.md              # Frequently asked questions
│   ├── misc.md             # Miscellaneous information
│   ├── usage.md            # Getting started guide
│   └── ...                 # etc.
└── ...
```

**Reply by ChinmayMhatre** [reply]
Was looking to implement this in my future projects. Thank you !!

**Reply by Ashutosh00710** [reply]
Go ahead, it will encourage new contributors to contribute to your project.
